### ISSUE TITLE
Make a "pathway" for the O_S_M targets

### ISSUE THREAD

**Issue by egonw** [OpenSourceMalaria/OSM_To_Do_List#300]
Tweets:

@egonwillighagen @cdsouthan @xanderpico @Chris_Evelo Hi gents - most OSM targets not known. Phenotypic discovery. Series 4 likely PfATP4 1/2
https://twitter.com/MatToddChem/status/610415110373294081

@egonwillighagen @cdsouthan @xanderpico @Chris_Evelo 2/2 but relationship also phenotypic - no proven binding interaction yet. No X-stal.
https://twitter.com/MatToddChem/status/610415395636277250

@MatToddChem @egonwillighagen @cdsouthan @xanderpico We can show genes in @WikiPathways involved in process but not known targets 1/2
https://twitter.com/Chris_Evelo/status/610426253145997313

@MatToddChem @egonwillighagen @cdsouthan @xanderpico We can also add a table of genes known to be targets with no known function 2/2
https://twitter.com/Chris_Evelo/status/610426390526230528

**Comment by cdsouthan** [comment]
Mapping of antimalarial chemotypes against data-supported molecular targets in Plasmodium biochemical pathways is obviosly valuable.  However, since the current OSM strategy is phenotypic, the utility will be more at the community level for now.  For example PMID 25322084 points out PfATP4 (a.k.a  UniProt Q8I5T3) is a target for aminopyrazole and spiroindolone chemotypes  (so get Series 4 tested against those mutants?)  and enonw has now mapped (an isomer of)  MMV008138   to Q8I273 (http://cdsouthan.blogspot.se/2015/05/antimalarial-dot-joining-for-mmv008138.html).  There are a lot of issues around Plasmodium pathway mapping that I can expand if there is interest,  one of them being that Swiss-Prot could do a full proteome curation (for D7 at least) to provide the best starting point.

**Comment by mattodd** [comment]
Hi gents. Yes - can I just clarify on a basic level - is this to try to find/validate the protein target of the Series 4 compounds? Is that the aim? It seems as though PfATP4 is the target, based on an ion regulation assay - which is to say that the Series 4 compounds have the same phenotypic effect as other compounds thought to be active against that target. The supposition is based on genome sequencing of resistant mutants, NOT an identified binding interaction. @cdsouthan we've done the screen vs some resistant mutants, supporting the idea that Series 4 has the same mech:

http://malaria.ourexperiment.org/biological_data/11448/Evaluation_of_Series_4_Compounds_vs_ATP4Resistant_Mutants.html

But it's always possible that this is not the protein target and that this effect on ion regulation is e.g. downstream of something else, or simply the most visible response to what is a different binding interaction. The diversity of structures that share this phenotypic effect is striking:

http://malaria.ourexperiment.org/the_osm_blog/11547/The_Mysterious_World_of_ATP4_Inhibitors.html

From my side I would like, in the next month, to re-start a pharmacophore modelling project we tried but which was not predictive of activity. More on that ASAP ( @murrayfold  ) , but in the meantime how does this "pathway" project fit in with target ID?

**Comment by murrayfold** [comment]
I've quickly added the pharmacophore work from last year here http://malaria.ourexperiment.org/pharmacophore_modelling_ 

I've not added the biological data as I thought that should maybe go elsewhere if it's not already up.  If it is already post, I can link to it.

**Comment by mattodd** [comment]
Legend. By "biological data" you mean the correspondence between prediction
and outcome in Kiaran Kirk's ion homeostasis assay?


On 16 June 2015 at 23:31, Murray Robertson <notifications@github.com> wrote:

>  I've quickly added the pharmacophore work from last year here
> http://malaria.ourexperiment.org/pharmacophore_modelling_
>
> I've not added the biological data as I thought that should maybe go
> elsewhere if it's not already up. If it is already post, I can link to it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenSourceMalaria/OSM_To_Do_List/issues/300#issuecomment-112429634>
> .
>



-- 
MATTHEW TODD | Associate Professor
School of Chemistry | Faculty of Science

THE UNIVERSITY OF SYDNEY
Rm 519, F11 | The University of Sydney | NSW | 2006
T +61 2 9351 2180  | F +61 2 9351 3329  | M +61 415 274104
E matthew.todd@sydney.edu.au | W
http://sydney.edu.au/science/chemistry/research/todd.html | W
http://opensourcemalaria.org/

CRICOS 00026A
This email plus any attachments to it are confidential. Any unauthorised
use is strictly prohibited. If you receive this email in error, please
delete it and any attachments.

**Comment by murrayfold** [comment]
Yeah, I know all the commercial ones came back inactive but I can't lay my hands on any info regards the assay or the conditions

**Comment by mattodd** [comment]
OK, leave with me, I'll dig up and post.

On 16 June 2015 at 23:46, Murray Robertson <notifications@github.com> wrote:

>  Yeah, I know all the commercial ones came back inactive but I can't lay
> my hands on any info regards the assay or the conditions
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenSourceMalaria/OSM_To_Do_List/issues/300#issuecomment-112435582>
> .
>



-- 
MATTHEW TODD | Associate Professor
School of Chemistry | Faculty of Science

THE UNIVERSITY OF SYDNEY
Rm 519, F11 | The University of Sydney | NSW | 2006
T +61 2 9351 2180  | F +61 2 9351 3329  | M +61 415 274104
E matthew.todd@sydney.edu.au | W
http://sydney.edu.au/science/chemistry/research/todd.html | W
http://opensourcemalaria.org/

CRICOS 00026A
This email plus any attachments to it are confidential. Any unauthorised
use is strictly prohibited. If you receive this email in error, please
delete it and any attachments.
